### PR TITLE
Updated 02_Statistics

### PR DESCRIPTION
**Pull request by abr-98**
Updated:

- EDA
- Histograms
- Percentiles and Histograms
- Probability Theory
- Bayes Theorem
- Random Variables
- Skewness

### CODE REVIEW

File: 02_Statistics/README.md [hunk 1]
```diff
@@ -50,22 +50,188 @@ The median is the value __separating the higher half of a data sample, a populat
...
+
+    import matplotlib.pyplot as plt
+
+[Pandas](https://pandas.pydata.org/)
+
+Library used to large datasets in python
+
+__Installation__:
+
+    sudo pip3 install pandas
```
Comment (MrMimic) [inline 02_Statistics/README.md:81]: No installation with sudo. Better to create a venv first.

File: 02_Statistics/README.md [hunk 2]
```diff
@@ -50,22 +50,188 @@ The median is the value __separating the higher half of a data sample, a populat
...
+
+    import pandas as pd
+    
+[Seaborn](https://seaborn.pydata.org/)
+
+Yet another Graph Plotting Library in Python.
+
+__Installation__:
+
+    sudo pip3 install seaborn
```
Comment (MrMimic) [inline 02_Statistics/README.md:93]: Idem here

**Review by MrMimic — CHANGES_REQUESTED** [review]
That's a great PR ! Thanks ! Juste few comments about the use of "sudo" while installling Python packages.

File: 02_Statistics/README.md (hunk 1 above)
Comment (abr-98) [reply 02_Statistics/README.md:81]: Yeah!! I missed that. Using "sudo" will install to the root. It's better to use a venv. Thanks

File: 02_Statistics/README.md (hunk 2 above)
Comment (abr-98) [reply 02_Statistics/README.md]: I have resolved the issue

**Comment by abr-98** [comment]
Updated:

- 8_ Random variables
- 9_ Cumul Dist Fn (CDF)
- 10_ Continuous distributions
- 12_ ANOVA
- 13_ Prob Den Fn (PDF)
- 16_ Hypothesis Testing
- 17_ p-Value
- 18_ Chi2 test
- 22_ Kernel Density estimate
- 23_ Regression
- 24_ Covariance
- 25_ Correlation
- 29_ Euclidian Distance

**Comment by abr-98** [comment]
Updated  04_Machine-Learning

updated:

- 2_ Numerical var
- 3_ Categorical var
- 4_ Supervised learning
- 5_ Unsupervised learning
- 6_ Concepts, inputs and attributes
- 7_ Training and test data
- 8_ Classifiers
- 9_ Prediction
- 10_ Lift
- 11_ Overfitting
- 12_ Bias & variance
- 13_ Tree and classification
- 14_ Classification rate
- 15_ Decision tree
- 16_ Boosting

**Comment by abr-98** [comment]
Added Reinforcement Learning

**Comment by abr-98** [comment]
Updated:

- 17_ Naïves Bayes classifiers
- 18_ K-Nearest neighbor
- 19_ Logistic regression
- 20_ Ranking
- 21_ Linear regression
- 23_ Hierarchical clustering
- 24_ K-means clustering
- 25_ Neural networks
- 26_ Sentiment analysis
- 27_ Collaborative filtering
- 28_ Tagging
- 29_ Support Vector Machine
- 30_Reinforcement Learning

**Review by MrMimic — APPROVED** [review]

File: 02_Statistics/README.md (hunk 2 above)
Comment (MrMimic) [reply 02_Statistics/README.md]: Sorry, was busy !

Thanks for this nice update !

**Pull request merged**